docs/using-az-aro.md

```diff
@@ -1,4 +1,4 @@
-# Using `az aro`
+# Developing with `az aro` extension
```

Comment:
I think this file should go away entirely.  The relevant dev extension configuration pieces should carefully be rolled into deploy-development-rp.md.